src/components/Card/style.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import { Colors } from '../../tokens'
+import { CardProps } from './type'
+
+const StyledCard = styled.div<CardProps>`
+  display: flex;
+  max-width: ${(props: CardProps) => props.maxWidth || '100%'};
+  padding: 16px;
+  gap: 16px;
```

Comment:
get sizes by `Spacing` tokens

Reply:
This space cannot be the `Spacing` token, it is in rem and needs to be pix